### PR TITLE
Feat/add time picker

### PR DESCRIPTION
**Pull request by Leon-pix-coder**
add time picker

### CODE REVIEW

File: src/components/SimpleDatePicker/index.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { DateRange, DayPicker } from "react-day-picker";
+import "react-day-picker/style.css";
+import Drawer from "../Drawer";
+import "./index.css";
+import { useState } from "react";
+
+interface BaseDatePickerProps {
+  isVisible?: boolean;
+}
```
Comment (evan-spidy) [inline src/components/SimpleDatePicker/index.tsx]: if this only use once you do not need to declare another and extends

File: src/components/SimpleTimePicker/index.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+  const handleConfirm = () => {
+    const selectTime = dayjs(
+      `${selectedHour}:${selectedMinute} ${selectedPeriod}`,
+      "hh:mm A"
+    );
+    onChange?.(selectTime.format("HH:mm"));
+  };
+
+  console.log(isVisible);
```
Comment (evan-spidy) [inline src/components/SimpleTimePicker/index.tsx]: remove console.log

File: src/components/SimpleTimePicker/index.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+dayjs.extend(customParseFormat);
+
+const hours = Array.from({ length: 12 }, (_, i) => `${i + 1}`);
+
+const minutes = Array.from(
+  { length: 60 },
+  (_, i) => `${i < 10 ? "0" : ""}${i}`
+);
+
+const periods = ["AM", "PM"];
```
Comment (evan-spidy) [inline src/components/SimpleTimePicker/index.tsx]: if it is a constants use ALL_CAPS

File: src/components/SimpleTimePicker/index.tsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+const SimpleTimePicker = ({ isVisible, onChange }: ISimpleTimePickerProps) => {
+  const [selectedHour, setSelectedHour] = React.useState(hours[0]);
+  const [selectedMinute, setSelectedMinute] = React.useState(minutes[0]);
+  const [selectedPeriod, setSelectedPeriod] = React.useState(periods[0]);
```
Comment (evan-spidy) [inline src/components/SimpleTimePicker/index.tsx]: avoid using React.useState, you already import React on top add it there

File: src/components/Vote/index.tsx [hunk 5]
```diff
@@ -22,6 +25,23 @@ const Vote = () => {
...
+        isVisible={false}
+        direction="bottom"
+        rootClassName="pt-[34px] pb-[40px] bg-tertiary"
+      >
+        <span className="block mb-[29px] text-[18px] font-outfit font-bold leading-[18px] text-center text-white">
+          Creating Poll
+        </span>
+        <img
+          className="mx-auto w-[236px] h-[208px] object-contain"
+          src=""
```
Comment (evan-spidy) [inline src/components/Vote/index.tsx]: this is empty

File: src/index.css [hunk 6]
```diff
@@ -15,6 +15,9 @@
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+
+  --primary: #8772FF;
```
Comment (evan-spidy) [inline src/index.css:19]: add it in tailwind also, look at how we add background and foreground in  tailwind

Comment (Leon-pix-coder) [reply src/index.css:19]: updated

File: src/components/Vote/index.tsx (hunk 5 above)
Comment (Leon-pix-coder) [reply src/components/Vote/index.tsx]: Updated

File: src/components/SimpleTimePicker/index.tsx (hunk 4 above)
Comment (Leon-pix-coder) [reply src/components/SimpleTimePicker/index.tsx]: Updated

File: src/components/SimpleTimePicker/index.tsx (hunk 3 above)
Comment (Leon-pix-coder) [reply src/components/SimpleTimePicker/index.tsx]: Updated

File: src/components/SimpleTimePicker/index.tsx (hunk 2 above)
Comment (Leon-pix-coder) [reply src/components/SimpleTimePicker/index.tsx]: Removed

File: src/components/SimpleDatePicker/index.tsx (hunk 1 above)
Comment (Leon-pix-coder) [reply src/components/SimpleDatePicker/index.tsx]: Updated

**Pull request merged**